pkg/utils/apiutil/serverapi/middleware.go

```diff
@@ -171,6 +171,7 @@ func (h *redirector) ServeHTTP(w http.ResponseWriter, r *http.Request, next http
 			return
 		}
 		clientUrls = append(clientUrls, targetAddr)
+		w.Header().Set(apiutil.ForwardToMicroServiceHeader, "true")
```

Comment:
Is it only used for testing?

Reply:
Yes, it is now.

Reply:
If it is not added to resp, is there any other way to tell whether it is processed by the PD server or the scheduling server?

Reply:
When debugging on a real cluster, it may also be useful to identify the origin of the server handling the request.